packages/nodes-base/nodes/Microsoft/Outlook/trigger/MessageDescription.ts

```diff
@@ -0,0 +1,154 @@
+import type { INodeProperties } from 'n8n-workflow';
+import { messageFields } from '../v2/helpers/utils';
+
+export const properties: INodeProperties[] = [
+	{
+		displayName: 'Output',
+		name: 'output',
+		type: 'options',
+		default: 'simple',
+		options: [
+			{
+				name: 'Simplified',
+				value: 'simple',
+			},
+			{
+				name: 'Raw',
+				value: 'raw',
+			},
+			{
+				name: 'Select Included Fields',
+				value: 'fields',
+			},
+		],
+	},
+	{
+		displayName: 'Fields',
+		name: 'fields',
+		type: 'multiOptions',
+		description: 'The fields to add to the output',
+		displayOptions: {
+			show: {
+				output: ['fields'],
+			},
+		},
+		options: messageFields,
+		default: [],
+	},
+	{
+		displayName: 'Filters',
+		name: 'filtersUI',
+		type: 'fixedCollection',
+		placeholder: 'Add Filters',
+		default: {},
+		options: [
+			{
+				displayName: 'Values',
+				name: 'values',
+				values: [
+					{
+						displayName: 'Filters',
+						name: 'filters',
+						type: 'collection',
+						placeholder: 'Add Filter',
+						default: {},
+						options: [
+							{
+								displayName: 'Filter Query',
+								name: 'custom',
+								type: 'string',
+								default: '',
+								placeholder: 'e.g. isRead eq false',
+								hint: 'Search query to filter messages. <a href="https://learn.microsoft.com/en-us/graph/filter-query-parameter">More info</a>.',
+							},
+							{
+								displayName: 'Has Attachments',
+								name: 'hasAttachments',
+								type: 'boolean',
+								default: false,
+							},
+							{
+								displayName: 'Folders to Exclude',
+								name: 'foldersToExclude',
+								type: 'multiOptions',
+								typeOptions: {
+									loadOptionsMethod: 'getFolders',
```

Comment:
This isn't defined in the `loadOptions` method so currently doesn't work.

Reply:
Fixed, sorry for the inconvenience